layouts/default/split_3x5_3/info.json

```diff
@@ -0,0 +1,49 @@
+{
+    "keyboard_name": "3x5+3 split layout",
+    "url": "",
+    "maintainer": "qmk",
+    "width": 13,
+    "height": 4.75,
+    "layouts": {
+        "LAYOUT_split_3x5_3": {
+            "layout": [
+                {"x":2, "y":0},
+                {"x":10, "y":0},
+                {"x":1, "y":0.125},
+                {"x":3, "y":0.125},
+                {"x":9, "y":0.13},
+                {"x":11, "y":0.13},
+                {"x":4, "y":0.25},
+                {"x":0, "y":0.375},
+                {"x":8, "y":0.38},
+                {"x":12, "y":0.38},
+                {"x":2, "y":1.0},
+                {"x":10, "y":1.0},
+                {"x":1, "y":1.125},
+                {"x":3, "y":1.125},
+                {"x":9.01, "y":1.13},
+                {"x":11.0, "y":1.13},
+                {"x":4, "y":1.25},
+                {"x":0, "y":1.375},
+                {"x":8, "y":1.38},
+                {"x":12, "y":1.38},
+                {"x":2, "y":2},
+                {"x":10, "y":2},
+                {"x":1, "y":2.125},
+                {"x":3, "y":2.125},
+                {"x":9, "y":2.13},
+                {"x":11, "y":2.13},
+                {"x":4, "y":2.25},
+                {"x":0, "y":2.375},
+                {"x":8, "y":2.38},
+                {"x":12, "y":2.38},
+                {"x":2.5, "y":3.25},
+                {"x":9.5, "y":3.25},
+                {"x":3.5, "y":3.5},
+                {"x":8.5, "y":3.5},
+                {"x":4.5, "y":3.75},
+                {"x":7.5, "y":3.75}
```

Comment:
The ordering is incorrect here. KLE orders by X/Y coordinates, whereas info.json goes by the actual ordering of the elements in the array. So sometimes you may need to reorder them after conversion - you can test this by turning on fast input in the Configurator and spamming a key.

I've also tweaked the KLE to be properly symmetrical and aligned on 0.125u intervals, which avoids the rounding errors in the converter.
```suggestion
                {"x":0, "y":0.25},
                {"x":1, "y":0.125},
                {"x":2, "y":0},
                {"x":3, "y":0.125},
                {"x":4, "y":0.25},

                {"x":7, "y":0.25},
                {"x":8, "y":0.125},
                {"x":9, "y":0},
                {"x":10, "y":0.125},
                {"x":11, "y":0.25},

                {"x":0, "y":1.25},
                {"x":1, "y":1.125},
                {"x":2, "y":1},
                {"x":3, "y":1.125},
                {"x":4, "y":1.25},

                {"x":7, "y":1.25},
                {"x":8, "y":1.125},
                {"x":9, "y":1},
                {"x":10, "y":1.125},
                {"x":11, "y":1.25},

                {"x":0, "y":2.25},
                {"x":1, "y":2.125},
                {"x":2, "y":2},
                {"x":3, "y":2.125},
                {"x":4, "y":2.25},

                {"x":7, "y":2.25},
                {"x":8, "y":2.125},
                {"x":9, "y":2},
                {"x":10, "y":2.125},
                {"x":11, "y":2.25},

                {"x":2.5, "y":3.25},
                {"x":3.5, "y":3.5},
                {"x":4.5, "y":3.75},

                {"x":6.5, "y":3.75},
                {"x":7.5, "y":3.5},
                {"x":8.5, "y":3.25}
```